fragments/functions.sh

```diff
@@ -166,17 +175,19 @@ downloadURLFromGit() { # $1 git user name, $2 git repo name
         filetype=$type
     fi
 
+    gitURL="$gitusername/$gitreponame/releases/tags/$gitReleaseTag"
+
     if [ -n "$archiveName" ]; then
-        downloadURL=$(curl -sfL "https://api.github.com/repos/$gitusername/$gitreponame/releases/latest" | awk -F '"' "/browser_download_url/ && /$archiveName\"/ { print \$4; exit }")
+        downloadURL=$(curl -sfL "https://api.github.com/repos/$gitURL" | awk -F '"' "/browser_download_url/ && /$archiveName\"/ { print \$4; exit }")
```

Comment:
The fallback method which doesn't use the API works fairly well. Any reason to prefer a call to the API first? (other than bandwidth considerations).

got API requests down to 3 though so one would have to install 20 github sourced packages to it the limit